demo/Demo/Textfields.elm

```diff
@@ -224,6 +225,32 @@ textfields model =
         ]
        """
     )
+  , ( "Expandable textfield"
+    , Textfield.render Mdl [11] model.mdl
+        [ Textfield.label "Expandable"
+        , Textfield.floatingLabel
+        -- NOTE: The id's must match
```

Comment:
Maybe we should change that; this situation is bound to come up again.

Maybe change option.id to apply only to things that have an id field in their Config, then delay turning that id into an attribute until some later point?

Don't know which point, though. It would suck if we have to apply it manually, and it wouldn't work with style then.

Maybe special-case id in the Property c m type? I normally wouldn't want that, but you could argue that id is like class in that it is (well, will be) a very common thing to both set, and inspect, like here?